pkgs/default.nix

```diff
@@ -55,4 +55,4 @@ let
     }) pkgsPaths
   );
 in
-finalPkgs
+finalPkgs // import ./aliases.nix { inherit finalPkgs pkgs; }
```

Comment:
This alias support still does not work at all (`nix eval .#vm.closure` shows this clearly). Please use the implementation I just pushed to `main` instead. Thanks!